edi_exchange_template_oca/migrations/18.0.1.1.0/pre-migrate.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2025 Camptocamp SA (http://www.camptocamp.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openupgradelib.openupgrade import logged_query
+
+from odoo.tools.sql import column_exists, create_column, table_exists
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    if not table_exists(cr, "edi_exchange_template_type_rel"):
```

Comment:
@yankinmax if it exists we shouldn't do anything, no?

Reply:
`README.rst` is updated by pre-commit